### DISCUSSION TITLE
Need to put comment inside of any parent tag

### DISCUSSION THREAD

**Discussion by danwood** [pointfreeco/swift-html#83] [Q&A]
I'm trying to construct an HTML document with an IE-specific directive in the `<head>` tag. It looks like a comment:
```
<!--[if IE]><meta http-equiv='X-UA-Compatible' content='IE=edge'><![endif]-->
```
Problem is, I can't put this comment in the `<head>` tag. I get this compile message:

- Type 'ChildOf<Tag.Head>' has no member 'comment'

It would be great to be able to put some kinds of nodes like `.comment`, also probably `.raw` nodes, at any place in a tree.

**Comment by mbrandonw** [comment]
Hey @danwood, thanks for the issue! 

This is definitely a bit of a blind spot for the library. It is possible to do this, but perhaps not super obvious. The trick is to use the `Node.raw` escape hatch, which allows you to render any string into html, but then to make it properly embed into a `<head>` tag you have to further wrap it in a `ChildOf<Tag.Head>` like so:

```swift
Node.html(
  .head(
    .init(.raw("<!--[if IE]><meta http-equiv='X-UA-Compatible' content='IE=edge'><![endif]-->"))
  )
)
```

It's not the prettiest thing, but it'll work. I think we could improve the docs for our escape hatches and perhaps even provide a better escape hatch API here.
